toolbox

```diff
@@ -693,6 +693,7 @@ create()
     no_hosts=""
     toolbox_profile_bind=""
     ulimit_host=""
+    media_path_bind=""
```

Comment:
Maybe call the variable `run_media_path_bind`?

There's another request about making the `/media` bind mount conditional because it doesn't exist on openSUSE. This way the variables won't conflict.

Reply:
Sure.